ios/browser/api/favicon/favicon_loader.mm

```diff
@@ -5,41 +5,50 @@
 
 #include "brave/ios/browser/api/favicon/favicon_loader.h"
 #include "base/strings/sys_string_conversions.h"
+#import "brave/ios/browser/favicon/brave_ios_favicon_loader.h"
+#import "brave/ios/browser/favicon/brave_ios_favicon_loader_factory.h"
 #include "components/favicon_base/favicon_types.h"
 #include "ios/chrome/browser/application_context.h"
 #include "ios/chrome/browser/browser_state/chrome_browser_state.h"
 #include "ios/chrome/browser/browser_state/chrome_browser_state_manager.h"
-#include "ios/chrome/browser/favicon/favicon_loader.h"
-#include "ios/chrome/browser/favicon/ios_chrome_favicon_loader_factory.h"
 #include "ios/chrome/common/ui/favicon/favicon_attributes.h"
 #import "ios/chrome/common/ui/favicon/favicon_constants.h"
 #import "net/base/mac/url_conversions.h"
 
+#include <memory>
```

Comment:
should be:
```
#include "brave/ios/browser/api/favicon/favicon_loader.h"

#include <memory>

#include "base/strings/sys_string_conversions.h"
...
```
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Fixed